faqs/does-eth-2-solve-high-gas-prices.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: Does ETH 2.0 solve high gas prices?
+title: Does the merge solve high gas prices?
 weight: 8.0
 attribution:
   -
     name:
     link:
----
\ No newline at end of file
+---
+No. The merge is limited in scope to changing the consensus mechanism. In practice it should not have any effect on the current user experience of Ethereum today. The other half of Eth2, sharding, is what will help improve gas prices. Currently sharding is considered to be a slightly lower priority by most of the Ethereum community.
```

Comment:
- I've been referring to Eth2 as ETH 2.0 throughout - so I'd like to do so here for consistency.
- A linked source for reading up on sharding would be great
- ditto if there's any backing for the statement of it being a lower priority (Github discussion, ETH Magicians post, etc)